udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,10 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Uses async and await to grab content from the data servlet.
```

Comment:
I'd suggest leaving out implementation details from documentation too, as that increases maintenance cost of the documentation if the implementation changes. Instead, you can keep it generic:

```
Asynchronously fetches comment content from the data servlet.
```